### DISCUSSION TITLE
Welcome to ARCs Discussions!

### DISCUSSION THREAD

**Discussion by howardwu** [ProvableHQ/ARCs#4] [Announcements]
## 👋 Welcome to Aleo ARCs!
  We’re using **Github Discussions** to facilitate conversations about ARCs with members of our developer community.

We hope that you:
  * Propose new ideas.
  * Ask questions.
  * Review ideas.
  * Engage with other developers.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by G2575869736** [comment]
致Aleo团队的一封信:
前言:
现在是2025年1月9日晚11时12分，从2024年8月份关注Aleo项目已有5个多月。以下说说自己的看法和感受，初次了解Aleo发展历程，自2019年底成立了Aleo项目，团队开发了很多底层架构和设施以支持Aleo长久发展。比如:SnarkOS，SnarkVM，Zk cloud
第一次在Aleo.org官网了解Aleo，觉得这个项目非常有前途，毕竟市场上现有的项目，都是千篇一律重复以太坊走过的道路，毫无新意。Aleo独树一帜，从设计理念完全脱离以太坊概念，到零知识证明的全面拓展，其中最耳目一新的就是Zk cloud链下计算技术。
2024年9月Aleo主网上线，并且在中心化交易所上市交易了，大家都欢呼雀跃。Aleo改变区块链发展瓶颈的机会终于到来了。但好景不长，Aleo在高位坚持了十天左右，然后币价一路阴跌至1.1美金附近，社区大量质疑声出现，矿工倒挂，挖矿产出完全不够支付电费……
有人在discord上发言咨询一些问题，然而都被踢了出来，Aleo社区甚至不能出现质疑的声音。试问，项目要发展，不能有任何质疑的声音出现，必须保证热情高涨。这种低迷情绪下，社区和矿工如何保持？
我个人的理解是:要允许出现不同的声音，有分歧太正常不过了，有问题解决问题，有意见探讨意见，最后达成共识就好。
随着11月至12月的到来，Aleo代币一路阴跌不止，连续创出30天跌幅，累计下跌90%。在市场上，无论是VC和交易员，看见Aleo这样的盘面，估计直接躲开了，因为Aleo是奔着归零去的。
真心希望Aleo团队能组织一下，找出一个靠谱的人，解决一下社区目前存在的问题，列出Aleo后续发展路线图，对于现有诸多问题避而不谈，问题不会解决，反而累积到最后，会成为拖死Aleo最后的那一颗稻草。
根据Aleo主网生态图和Aleo盘面来说，Aleo团队对技术发展有优势，对零知识证明有深刻的理解和研究，生态发展在稳步进行当中，唯一的问题是生态使用人数较少，隐私保护对于大多数人来讲似乎可有可无，但是随着时间的推移，隐私保护问题会被越来越多人提及，这也是Aleo正在做的事情。
根据Aleo盘面和交易来看，Aleo没有做市商，也没有市值管理，甚至团队对于价格走势毫不在意，任其发展。
很多项目都是技术和盘面结合的结果，并不是一味的发展技术。技术是项目发展的底层储备和长期发展的基石，这一点非常认可。但是，市场认可也非常重要，以太坊主要开发人V神公开提及过，他给以太坊的定位是服务于社会的公共区块链，并不是市场和炒作。但是以太坊要发展，必须要有社区和市场支持才行，否则只能走向灭亡。
ETH每次升级，都是围绕市场支持和社区，V神对以太坊有完美的构想，但是他得不以社区和市场为中心。因为他明白，让社区和市场获利，以太坊才有发展的机会。
中国有句古话:天下熙熙，皆为利来，天下攘攘，皆为利往。这个世界的任何人，任何事都是因利而起，人无利不往，无往不利。
BTC若是只有技术供应，没有强大的代币经济学对矿工的激励和交易带来的实际利益，恐怕现在区块链技术还是停留在十六年以前。中本聪很聪敏，他明白，只有利益的激励，围绕比特币的区块链技术才能推广到市场，最后实现商用，造福人类社会。
Aleo想要发展，必须得到社区和市场的支持，否则只靠纯技术推进，没有社区和市场加持，Aleo美好的设想也无法实现，因为被社区和市场诟病的项目不会有好的发展。
我们尊重技术，也感谢零知识证明给区块链带来的发展和机遇。希望Aleo团队在后续的发展中，抽出时间听听社区的声音，了解市场发展实际情况，开发项目并不是闭门造车，各方的意见也需要采纳。     
以上，感谢倾听！！！
2025年1月9日晚

**Reply by howardwu** [reply]
Thank you for the constructive feedback -- both praises and critiques on Aleo. The roadmap for Aleo remains long-term oriented, and there's a growing need to focus beyond the technology itself, namely on the ecosystem & community and the evolving market landscape. With this new year and changes in leadership, let's prioritize addressing and correcting these points with the community at large.

**Reply by G2575869736** [reply]
感谢回复，2025年Aleo会越来越好！

**Comment by G2575869736** [comment]
关于Aleo生态方面么咨询:
Aleo目前主网生态包括:质押、钱包、NFT、DEX、跨链桥。缺失的生态有金融借贷和预言机平台，网上有关于Aleo借贷协议DinarMaker的介绍，还有Aleo预言机系统0rbit的介绍，请问这俩个板块的生态什么时候上线主网，补齐Aleo常规生态的拼图。

另外，在官网上看到有Zk ML(零知识和机器证明)的相关介绍，Aleo的隐私性可以为医疗等行业发挥很大优势，并且目前区块链发展逐渐靠拢AI，Aleo官方或者开发团队对于AI是否有相关计划和安排，让AI助力于Aleo生态快速发展，打造Aleo护城河！感谢！！！
